numpy/core/_add_newdocs.py

```diff
@@ -5652,15 +5649,11 @@
         byte order.  `new_order` codes can be any of:
 
         * 'S' - swap dtype from current to opposite endian
-        * {'<', 'L'} - little endian
-        * {'>', 'B'} - big endian
-        * {'=', 'N'} - native order
+        * {'<'} - little endian
+        * {'>'} - big endian
+        * {'='} - native order
         * {'|', 'I'} - ignore (no change to byte order)
```

Comment:
Seems inconsistent to use sets to describe things with one element.

Reply:
sure good point, will change this